cloud-sql/mysql/mysql/package.json

```diff
@@ -10,51 +9,25 @@
     "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git"
   },
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=10.0.0"
```

Comment:
I won't argue, though technically 8 ain't dead yet :P

Reply:
It's depreacted on GAE since October 1st. https://cloud.google.com/appengine/docs/standard/nodejs/runtime